### PR TITLE
SAAS-259 updated selectors and methods for Upgrade job

### PR DESCRIPTION
**Pull request by yurkovychv**
[SAAS-259](https://jira.percona.com/browse/SAAS-259)

Updated selectors for different Upgrade methods
Fixed typo in one of data-qa attributes

### CODE REVIEW

**Comment by puneet0191** [comment]
good! work thanks a lot

File: pmm-app/tests/pages/homePage.js [hunk 1]
```diff
@@ -13,68 +14,91 @@ module.exports = {
...
     newsPanelContentSelector: "//span[contains(text(), 'Percona News')]/ancestor::div[contains(@class, 'panel-container')]//div[contains(@class, 'view')]",
     noAccessRightsSelector: '$db-check-panel-no-access',
+    updateWidget: {
+      base: {
+        checkUpdateButton: "#refresh",
+        updateProgressModal: ".modal-content",
+        successUpgradeMsgSelector: ".modal-content",
+        lastCheckSelector: ".last-check-wrapper > p",
+        triggerUpdate: "button[ng-click='update()']",
+        reloadButtonAfterUpgrade: "button[ng-click='reloadAfterUpdate()'",
```
Comment (atymchuk) [inline pmm-app/tests/pages/homePage.js:32]: Is the closing bracket missing, or is it intentional? Question is how does it work then?

File: pmm-app/tests/pages/homePage.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
-const {I, settingsAPI} = inject();
+const {I} = inject();
 const assert = require('assert');
+const lastCheckRegex = /^(?:(((Jan?|Ma(r?|y)|Jul(y)?|Aug?|Oct?|Dec?)\ 31)|((Jan?|Ma(r?|y)|Apr?|Ju((ly?)|(ne?))|Aug?|Oct?|(Sep(t)?|Nov|Dec)?)\ (0?[1-9]|([12]\d)|30))|(Feb?\ (0?[1-9]|1\d|2[0-8]|(29(?=,\ ((1[6-9]|[2-9]\d)(0[48]|[2468][048]|[13579][26])|((16|[2468][048]|[3579][26])00)))))))\,) (?:[0-1]?[0-9]|[2][1-4]):[0-5]?[0-9]?\s??$/gim;
```
Comment (atymchuk) [inline pmm-app/tests/pages/homePage.js:3]: What are all those magic numbers, like `[2468][048]|[13579][26]` ? Can we link this regexp to an eventual source (in a comment maybe)?

File: pmm-app/tests/pages/homePage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/pages/homePage.js]: For some reason it worked actually, but fixed now. Thank you

File: pmm-app/tests/pages/homePage.js (hunk 2 above)
Comment (yurkovychv) [reply pmm-app/tests/pages/homePage.js]: Great point. Sorry, looks like i did not cleanup that regex completely. Added the source link to the comment

**Pull request merged**